google-basemap/.env

```diff
@@ -0,0 +1,9 @@
+# API Base URL (copy this from CARTO Workspace -> Developers)
+VITE_API_BASE_URL=https://gcp-us-east1.api.carto.com
+# Connection ID (app.carto.com -> Developers -> Manage API Access Tokens)
+VITE_API_CONNECTION_NAME=bqconn2
+# Set this variable if you're using an API Access Token (public application).
+# Go to app.carto.com -> Developers -> Manage API Access Tokens
+VITE_API_ACCESS_TOKEN=eyJhbGciOiJIUzI1NiJ9.eyJhIjoiYWNfN3hoZnd5bWwiLCJqdGkiOiIwMDEzOTI5NCJ9.Y06WtEddE6a-CoGuad8AwMgWV26pC5OowXKxzUOcXIg
+VITE_GOOGLE_API_KEY=AIzaSyAVqQgxcffQME2Hsid6_JNBefdeRHO_7ns 
+VITE_GOOGLE_MAP_IDS=
```

Comment:
I would like to add a Basemap selector in this example to allow the user to change the basemap. The list of maps should be the same than builder

Reply:
Ok, wip

Reply:
Done